server/database/migrations/2016_08_11_144504_clean_up_duplicate_individual_groups.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+use App\Models\Group;
+
+class CleanUpDuplicateIndividualGroups extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        // Get unique collection of individual group names
+        $individual_groups = Group::select('name', 'id')->where('individual', 1)->orderBy('id', 'ASC')->groupBy('name')->get();
+
+        foreach ($individual_groups as $group) {
+            // Check if another individual group exists under same name
+            $duplicate_group = Group::where('name', $group['name'])
```

Comment:
Is it possible for multiple duplicates of the same group to exist? Because we only handle the first one if that's the case.